src/goto-instrument/dump_c_class.h

```diff
@@ -74,6 +86,46 @@ struct dump_c_configurationt final
   /// The config used for dump-c-type-header
   static dump_c_configurationt type_header_configuration;
 
+  dump_c_configurationt disable_include_functions_decl(bool value=false){
```

Comment:
If it's named `disable_...` then I wouldn't expect a parameter that could actually cause the opposite effect.